color.cpp

```diff
@@ -214,6 +215,33 @@ void test_color()
   }
 #endif // FIX_ISSUE_460
 
+#ifdef FIX_ISSUE_628
+  // (628) A random color between red, blue, green can be sampled
+  {
+    color c;
+    int n_red = 0;
+    int n_green = 0;
+    int n_blue = 0;
+
+    const color r = create_red_color();
+    const color g = create_green_color();
+    const color b = create_blue_color();
+
+    std::mt19937 rng(1);
+    for (int i = 0; i < 100; ++i)
+      {
+        c = get_random_rgb(rng);

```

Comment:
One could write `const color c = ...` here. It is a detail, but let's show the newbies our best, shall we :-) ?

Reply:
Isn't it going to be an issue when I change the value of `c` in the for loop? I thought `const` was only for objects that don't change value after being declared

Reply:
Yes, your believes are correct! The compiler should not complain when making this `const`. Or did I miss something???

Reply:
Ok, I got this, I'll just remove the declaration above and instead declare `c` inside the for loop